articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,525 @@
+---
+title: Azure Application Insights for ASP.NET Core applications | Microsoft Docs
+description: Monitor ASP.NET Core web applications for availability, performance, and usage.
+ms.topic: conceptual
+ms.devlang: csharp
+ms.custom: devx-track-csharp
+ms.date: 10/12/2021
+ms.reviewer: casocha
+---
+
+# Application Insights for ASP.NET Core applications
+
+This article describes how to enable Application Insights for an [ASP.NET Core](https://docs.microsoft.com/aspnet/core) application.
+
+Application Insights can collect the following telemetry from your ASP.NET Core application:
+
+> [!div class="checklist"]
+> * Requests
+> * Dependencies
+> * Exceptions
+> * Performance counters
+> * Heartbeats
+> * Logs
+
+We'll use an [ASP.NET Core MVC application](/aspnet/core/tutorials/first-mvc-app) example that targets `net6.0`. You can apply these instructions to all ASP.NET Core applications. If you're using the [Worker Service](/aspnet/core/fundamentals/host/hosted-services#worker-service-template), use the instructions from [here](./worker-service.md).
```

Comment:
lets also state upfront that this tutorial focuses on asp.net core app deployed as Azure Web App.

Reply:
lets also be upfront that this uses SDK based approach, and advice users about the existence of the auto-instrumentation as well (with links to it)

Reply:
Fixed.